Resources/variables.robot

```diff
@@ -1,54 +1,59 @@
-${InputNameTextField}  //*[@id="app"]/div[2]/div/div/div[2]/div/div[4]/div/form/div[1]/div/div/div[1]/div/div/div[1]/div/input
+${DeleteOption}  //*[@class="v-icon icon-menu-danger material-icons theme--light"]
+
+
+# Model
+${ContinueCreateModelButton}  //button[contains(.,'Continue')]
+${InputNameTextField}  //input[contains(@aria-label,'Name*')]
 ${ModelName}   Data Model
-${InputDescriptionTextField}  //*[@id="app"]/div[2]/div/div/div[2]/div/div[4]/div/form/div[1]/div/div/div[2]/div/div/div[1]/div/textarea
+${InputDescriptionTextField}  //*[@name="input-7-1"]
 ${ModelDescription}  This is A Model
-${CreateModelButton}  //*[@id="app"]/div[2]/div/div/div[2]/div/div[4]/div/form/div[2]/button
-${SingleModelOptionsButton}  //*[@id="app"]/div[7]/div[1]/main/div/div/div[3]/div/div/div/div/div/nav/div/div[3]/div/button/div/i
-${DeleteOption}  //*[@class="v-icon icon-menu-danger material-icons theme--light"]
-${StartTrainingButton}  //*[@id="app"]/div[7]/div[1]/main/div/div/div[2]/div/div[1]/div[2]/div/a/div
-${OverviewButton}  //*[@id="app"]/div[7]/div[1]/main/div/div/div[3]/div/div/div/div/div/div[2]/a/div
-${AddALabelButton}  //*[@id="app"]/div[4]/div/div[1]/div[4]/div/div/span/div/div/div[2]/div/div/button/div
-${AddLabelButton}  //*[@id="app"]/div[5]/div/div[1]/div[4]/div/div/span/div/div[2]/form/div[2]/button
+${CreateModelButton}  //button[contains(.,'Create Model')]
+${StartTrainingButton}  //*[contains(text(),'model_training')]
+${SliceFakeNews}  css:div#plotly > div.plot-container.plotly > div > svg:nth-child(1) > g.pielayer > g > g:nth-child(3) > g
+${SliceEconomy}   css:div#plotly > div.plot-container.plotly > div > svg:nth-child(1) > g.pielayer > g > g:nth-child(1) > g
+${SliceSport}  css:div#plotly > div.plot-container.plotly > div > svg:nth-child(1) > g.pielayer > g > g:nth-child(2) > g
+
+#Dataset
+${DataSelectCustomerSupportButton}  //*[@id="app"]/div[2]/div/div/div[2]/div/div[2]/div/div[3]/div/div[2]/div/div/div[2]/button/div
+${DataSelectCustomerResponseButton}  //*[@id="app"]/div[2]/div/div/div[2]/div/div[2]/div/div[2]/div/div/div/div/div[2]/button/div
+${ContinueWithSelectedDatasetColumnButton}  //*[@id="app"]/div[2]/div/div/div[2]/div/div[3]/div/div/div[2]/button
+
 
-${FirstRadioRoundButton}  //*[@id="app"]/div[8]/div/div[1]/div[4]/div/div/span/div/div/div/div[2]/div/div[1]/a/div[1]/i
-${SecondRadioRoundButton}  //*[@id="app"]/div[8]/div/div[1]/div[4]/div/div/span/div/div/div/div[2]/div/div[2]/a/div[1]/i
+#Label Training
+${AddALabelButton}  //*[@id="app"]/div[4]/div/div[1]/div[4]/div/div/span/div/div/div[2]/div/div/button/div   #Adds new label
+${AddLabelButton}  //*[@id="app"]/div[5]/div/div[1]/div[4]/div/div/span/div/div[2]/form/div[2]/button  #Adds label when you input label Name
+${LabelsPieChart}  //*[@id="plotly"]/div[1]/div
+${MultipleChoiceButton}  //button[contains(.,'Multiple Choice')]
+${AddButton}  //button[contains(.,' Add ')]

```

Comment:
Maybe This could be named "FinishAddLabelButton" or is it used in more contexts than to Confirm add label?